src/Compilers/Core/Portable/DiagnosticAnalyzer/DefaultAnalyzerAssemblyLoader.cs

```diff
@@ -12,6 +12,10 @@
 using System.Runtime.InteropServices;
 using Roslyn.Utilities;
 
+#if NET
```

Comment:
Nit: Inconsistent usage of NET or NETCOREAPP within the same file

Reply:
@jaredpar  what's your preference here.  IDE uses NET for this.  not sure how it differs from NETCOREAPP.

Reply:
The code base largely uses `NETCOREAPP` for historical reasons. The more idiomatic approach is `#if NET`.